src/isomorphic/modern/class/ReactComponent.js

```diff
@@ -66,14 +65,6 @@ ReactComponent.prototype.setState = function(partialState, callback) {
     'setState(...): takes an object of state variables to update or a ' +
     'function which returns an object of state variables.'
   );
-  if (__DEV__) {
-    ReactInstrumentation.debugTool.onSetState();
-    warning(
-      partialState != null,
-      'setState(...): You passed an undefined or null state object; ' +
-      'instead, use forceUpdate().'
-    );
-  }
   this.updater.enqueueSetState(this, partialState);
```

Comment:
`this.updater` falls back to the `ReactNoopUpdateQueue`which is also in `isomorphic/modern/class/`. Is it okay to not call `ReactInstrumentation.debugTool.onSetState()` in that scenario?